ext/sqlite3/statement.c

```diff
@@ -447,9 +587,10 @@ init_sqlite3_statement(void)
     rb_define_method(cSqlite3Statement, "column_name", column_name, 1);
     rb_define_method(cSqlite3Statement, "column_decltype", column_decltype, 1);
     rb_define_method(cSqlite3Statement, "bind_parameter_count", bind_parameter_count, 0);
-    rb_define_private_method(cSqlite3Statement, "prepare", prepare, 2);
-
-#ifdef HAVE_SQLITE3_COLUMN_DATABASE_NAME
-    rb_define_method(cSqlite3Statement, "database_name", database_name, 1);
+    rb_define_method(cSqlite3Statement, "stmt_stat", stmt_stat, 1);
```

Comment:
I think we should probably make this private, especially if we have the Ruby level `stat` method.

Reply:
Done in 089d00e